react/hooks/useScroll.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react'
+import throttle from 'lodash.throttle'
```

Comment:
I'd prefer to use a library like `debounce` instead of lodash. I think the resulting bundle may be bigger due to lodash's size.